packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -32,14 +130,86 @@ export default class AwsS3Multipart extends BasePlugin {
     this.#client = new RequestClient(uppy, opts)
 
     const defaultOptions = {
-      timeout: 30 * 1000,
+      timeout: 30 * 1000, // todo this in no longer in use. either implement or remove.
```

Comment:
It was added in https://github.com/transloadit/uppy/commit/3d8026340e95a1840049d3516a89552f305bce5d#diff-b98339cc15d96377452ab09e762ef118a7a504836f299f3aec139580472c2ac2R21, and has never been used oO
```suggestion
```